sharepoint/sharepoint-ps/Microsoft.Online.SharePoint.PowerShell/Get-SPOM365AgentAccessInsightsReport.md

```diff
@@ -0,0 +1,181 @@
+---
+external help file: Microsoft.Online.SharePoint.PowerShell.dll-Help.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-SPOM365AgentAccessInsightsReport 
+applicable: SharePoint Online
+title: Get-SPOM365AgentAccessInsightsReport
+schema: 2.0.0
+author: gchaudhary
+ms.author: gchaudhary
+ms.reviewer:
+manager: lokeshgoel
+---
+
+# Get-SPOM365AgentAccessInsightsReport

```

Comment:
>M365Agent [](http://example.com/codeflow?start=9&length=9)

what's the official name for this? As it's certainly not "M365 Agent" -- could you share a link showing the official documentation and naming?

Reply:
@starringGTM I don't have access to that doc. Is this documented publicly somewhere in learn.microsoft.com?